includes/elements/column.php

```diff
@@ -822,6 +822,8 @@ protected function register_controls() {
 			]
 		);
 
+		Plugin::$instance->controls_manager->add_display_conditions_controls( $this );
```

Comment:
What happens with this control in the Pro version?

Reply:
@hein-obox I have a PR to remove promotion control in PRO
https://github.com/elementor/elementor-pro/pull/3395